.ci/windows/msbuild-tools.ps1

```diff
@@ -3,6 +3,6 @@
 #
 
 # Install visualstudio2019buildtools
-& choco install visualstudio2019buildtools -m -y --no-progress --force -r --version=16.4.0.0
-& choco install visualstudio2019enterprise -m -y --no-progress --force -r --version=16.4.0.0 --package-parameters "--includeRecommended --includeOptional"
-& choco install visualstudio2019-workload-netweb -m -y --no-progress --force -r --version=1.0.0
+& choco install visualstudio2019buildtools -m -y --force --no-progress -r
+& choco install visualstudio2019professional -m -y --force --no-progress -r --package-parameters "--includeRecommended --includeOptional"
+& choco install visualstudio2019-workload-netweb -m -y --no-progress -r --version=1.0.0
```

Comment:
Let's add `--force` here, too. Maybe that'd help!

Reply:
Oh, I see you already have it in https://github.com/elastic/apm-agent-dotnet/pull/759 - that's good.